### PR TITLE
Extra comments and code refinement to better illustrate the interacti…

### PR DESCRIPTION
**Pull request by topcon-pconlon**
…on of file and design object handling with RDM.

### CODE REVIEW

File: components/files/file_features/file_features.py [hunk 1]
```diff
@@ -47,21 +54,37 @@ def query_file_features(a_server_config, a_site_id, a_file_upload_uuid, a_file_n
...
         features_to_import.append(v)
     return features_to_import
 
 
+
+
+# This function runs a process in the design file microservice called an "import job".
+# Import jobs perform the following steps:
+# 1. Import a list of design objects (which can be determined by the feature job in query_file_features) into Sitelink3D v2. 
+# 2. Add RDM representations for the imported design objects so that they can be found and referenced within Sitelink3D v2.
```
Comment (Ryan-A-B) [inline components/files/file_features/file_features.py:72]: The designfile service doesn't create the RDM objects, the client does. I can see the RDM payload being produced (line 132) but it doesn't look like anything is been done with it.

File: components/metadata/metadata_traits.py [hunk 2]
```diff
@@ -26,6 +26,9 @@ class FileMetadataTraits(MetadataTraitsBase):
     def __init__(self, a_object_value, a_display_name):
         MetadataTraitsBase.__init__(self, a_object_value, a_display_name)
 
+    # This function builds a bean that is posted to RDM for the purpose of representing a file.
+    # The _id field represents the file entity/object and hence will be the same for multiple revisions of the same file.
+    # This bean is not concerned with file upload identifiers. See FileUploadbean class in file_upload.py.
```
Comment (Ryan-A-B) [inline components/metadata/metadata_traits.py:31]: Not sure what you mean by this, the bean contains the upload-uuid in the "uuid" field

Comment (Ryan-A-B) [reply components/metadata/metadata_traits.py:31]: or the "sitelink_file_id" field

File: components/files/file_features/file_features.py (hunk 1 above)
Comment (topcon-pconlon) [reply components/files/file_features/file_features.py:72]: Sorry not sure what you mean by nothing being done with it. This payload is encoded at line 147 and posted at 150.

File: components/metadata/metadata_traits.py (hunk 2 above)
Comment (topcon-pconlon) [reply components/metadata/metadata_traits.py:31]: Good find. Misplaced comment.

**Review by Ryan-A-B — APPROVED** [review]

**Pull request merged**